strings/tcg/set_two.json

```diff
@@ -237,7 +237,7 @@
 			"id": "defiler",
 			"name": "Xenomorph Defiler",
 			"desc": "Instead of utilizing eggs, the Defiler prefers to inject an unknown chemical in their victim, causing a devastating infection.",
-			"rules": "{$Hivemind}. When this card attacks a target enemy creature, calculate damage as though the target creature has this card's power subtracted from it first.",
+			"rules": "{$Hivemind}. All enemy creatures lose 1 power.",
```

Comment:
Not entirely clear if it repeats or is just a static effect that goes away on death.